packages/rest/src/rest-application.ts

```diff
@@ -0,0 +1,39 @@
+import {Application, ApplicationConfig, Server} from '@loopback/core';
+import {RestComponent} from './index';
+import {Binding, Constructor} from '@loopback/context';
+import {format} from 'util';
+
+export const ERR_NO_MULTI_SERVER = format(
+  'RestApplication does not support multiple servers!',
+  'To create your own server bindings, please extend the Application class.',
+);
```

Comment:
nitpick: looks like an extra `,` here

Reply:
No, that's the coding style to have trailing `,` after the last argument if multiple lines are used.